auction_seller_invoice/views/auction_item_invoice_views.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- pylint:disable=deprecated-data-xml-node -->
```

Comment:
Do we need this line?

Reply:
Yes.  Pylint doesn't allow replace in xml file. So, I want to avoid pre-commit and added this line.